### PR TITLE
Update copyright notice for 2023

### PR DESCRIPTION
**Pull request by sequba**
### How did you test your changes?

unit test passed

### Types of changes

- [x] Other

### Checklist:

- [ ] My change is compliant with the [OpenDocument](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html) standard.
- [ ] My change is compatible with Microsoft Excel.
- [ ] My change is compatible with Google Sheets.
- [ ] My code follows the code style of this project.
- [ ] I described my changes in the [CHANGELOG.md](https://github.com/handsontable/hyperformula/blob/master/CHANGELOG.md) file.
- [ ] My changes require a documentation update.
- [ ] My changes require a migration guide.

### CODE REVIEW

File: test/emitting-events.spec.ts [hunk 1]
```diff
@@ -1,7 +1,11 @@
-import {ExportedCellChange, ExportedNamedExpressionChange, HyperFormula} from '../src'
-import {ErrorType} from '../src/Cell'
+import {
```
Comment (adamu-handsontable) [inline test/emitting-events.spec.ts:1]: Is this change supposed to be present in this PR? It seems unrelated

Comment (sequba) [reply test/emitting-events.spec.ts:1]: Yes, it is unrelated, but this is a unit test that I created while working on answer to this discussion: https://github.com/handsontable/hyperformula/discussions/1152

It's beneficial to keep this test in the codebase. Do you think I should have created a separate PR for adding it?

Comment (adamu-handsontable) [reply test/emitting-events.spec.ts:1]: Yes, let's split it to 2 separate PRs, just for clarification. The PR clearly states it's just a copyright notice, but new test was added as well.

**Review by adamu-handsontable — CHANGES_REQUESTED** [review]

File: test/emitting-events.spec.ts (hunk 1 above)
Comment (sequba) [reply test/emitting-events.spec.ts]: Ok, I removed this unit test from this branch

**Review by adamu-handsontable — APPROVED** [review]
Thank you, all is good.
